tests/test_integrated_channels/test_cornerstone/test_exporters/test_learner_data.py

```diff
@@ -220,6 +221,18 @@ def test_transmit_single_learner_data_performs_only_one_transmission(self):
             json={"is_passing": "true", "created_date": "2019-06-21T12:58:17.428373Z", "grade": "0.8"},
         )
 
+        responses.add(
```

Comment:
I have no idea how this requests call wasn't failing before...

Reply:
Actually was it failing in your PR before you added this section?
I imagine it wasn't failing previously due to the fact it seems to to fake a transmission audit in the setup:

From line 85 in this same file:
```
factories.CornerstoneLearnerDataTransmissionAuditFactory(
            user_id=user.id,
            session_token=self.session_token,
            callback_url=self.callback_url,
            subdomain=self.subdomain,
            course_id=course_key,
            user_guid=self.user_guid
        )
```

Reply:
Yeah, I think there's something here that I'm missing. Unless the fake transmission audit somehow prevents the actual  transmission or the client from being created, I don't see how this passed. I would have hoped that it would be caught by tests... like this one.

Reply:
you know what, we mock other client calls in other tests with the wrapper: https://github.com/edx/edx-enterprise/blob/master/tests/test_integrated_channels/test_cornerstone/test_exporters/test_learner_data.py#L133 I feel like that's going to be a little cleaner but now I'm wondering why it wasn't added to this one

Reply:
just kidding, if we make the client a magic mock like the above tests then it breaks when we check response bodies after the transmission. My first fix is still 👍  to me.